frontend/src/pages/detail/components/TitleSection/TitleSection.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import * as S from "./TitleSection.styled";
+import emptyHeartIcon from "../../../../assets/icons/heart-outline.svg";
+import heartIcon from "../../../../assets/icons/heart.svg";
+import githubIcon from "../../../../assets/icons/logo-github.svg";
+import linkIcon from "../../../../assets/icons/home.svg";
```

Comment:
```suggestion
import emptyHeartIcon from "@assets/icons/heart-outline.svg";
import heartIcon from "@assets/icons/heart.svg";
import githubIcon from ".@assets/icons/logo-github.svg";
import linkIcon from "@assets/icons/home.svg";
```

아마 다른 파일들은 절대 경로로 되어 있을 것 같은데 여기 파일만 상대 경로로 될 것 같아서 일관성 있게 절대 경로로 가도 될 것 같습니다~

Reply:
[refactor: 버튼 border color, asset 경로 수정](https://github.com/woowacourse-teams/2025-moaon/pull/39/commits/952dc06a64e86cb1f812f351ad6a39dc81c65cd8)